cvmfs/server/cvmfs_server_abort.sh

```diff
@@ -65,7 +65,9 @@ cvmfs_server_abort() {
     # more sanity checks
     is_owner_or_root $name || { echo "Permission denied: Repository $name is owned by $user"; retcode=1; continue; }
     check_repository_compatibility $name
-    is_in_transaction $name || { echo "Repository $name is not in a transaction"; retcode=1; continue; }
+    if [ x"$upstream_type" = xgw ]; then
```

Comment:
This would avoid the `is_in_transaction` test for the local/s3 backend.  Is this intended?